app/services/CountryService.scala

```diff
@@ -11,10 +11,12 @@ trait CountryService {
 
   val GB: Country
 
-  def findAll: Seq[Country]
+  val GBcy: Country
```

Comment:
did this need to be created?. it looks like the GB is used in just tests?.. both could be removed...

Reply:
I removed both vals GB and GBcy